### DISCUSSION TITLE
Welcome to dada-mail Discussions!

### DISCUSSION THREAD

**Discussion by justingit** [justingit/dada-mail#1131] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by admorgan** [comment]
I use Dada to allow my wife to send newsletters to her fans. I maintain the computer network for a multitude of organizations in addition to to her business. I utilize the integration with wordpress to present a consistent theme between Dada and the rest of her website. If you are interested in what she writes they can be found at https://www.amazon.com/Christine-Amsden/e/B001JOXSIW or any bookstore.

**Reply by justingit** [reply]
That's awesome!
